index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Kao Jai Coffee</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+
+<!-- Kao Jai Coffee Medium Roast Box -->
+    <article class="medium">
+      <header>
+        <h1>Kao Jai: Medium Roast</h1>
+      </header>
+
+        <section>
+          <img src="https://images-na.ssl-images-amazon.com/images/I/51YtolsHGQL.jpg">
```

Comment:
So this totally works and is fine. But technically the requirement asks that 'The first section should contain three child block elements.' and 5-i asks 'The first element contains the product image.'
You don't have to change it if you don't want, but it would be great practice to put your images in a block level element and then try to recenter them